infra/cifuzz/package.json

```diff
@@ -5,6 +5,6 @@
   "author": "Google",
   "license": "Apache2",
   "dependencies": {
-    "@actions/artifact": "1.1.1"
+    "@actions/artifact": "^3.1.0"
```

Comment:
Isn't the latest version 2.2.2?

Reply:
Thanks, I'm not sure where this came from.